huntflow_api_client/entities/webhooks.py

```diff
@@ -0,0 +1,24 @@
+from huntflow_api_client.entities.base import (
+    BaseEntity,
+    CreateEntityMixin,
+    DeleteEntityMixin,
+    ListEntityMixin,
+)
+from huntflow_api_client.models.request.webhooks import WebhookRequest
+from huntflow_api_client.models.response.webhooks import WebhookResponse, WebhooksListResponse
+
+
+class Webhook(BaseEntity, ListEntityMixin, CreateEntityMixin, DeleteEntityMixin):
+    async def list(self, account_id: int) -> WebhooksListResponse:  # noqa A003
```

Comment:
I'm sure we need to add the error `A003` to the ignore list, because class attributes and methods can have names similar to builtins
Please add `ignore = A003` to file `.flake8` and remove `# noqa A003` comment